packages/cohttp/cohttp.0.19.2/opam

```diff
@@ -40,7 +40,7 @@ depends: [
 ]
 depopts: ["async" "lwt" "js_of_ocaml"]
 conflicts: [
-  "async" {< "111.25.00"}
+  "async" {< "111.25.00" | >= "113.00.00"}
```

Comment:
On Thu, 3 Sep 2015 10:54:57 -0700, Thomas Gazagnaire wrote:
> 
> In packages/cohttp/cohttp.0.19.2/opam:
> 
> > @@ -40,7 +40,7 @@ depends: [
> >  ]
> >  depopts: ["async" "lwt" "js_of_ocaml"]
> >  conflicts: [
> > -  "async" {< "111.25.00"}
> > +  "async" {< "111.25.00" | >= "113.00.00"}
> 
> or a & ...

Or two lines so that there is no ambiguity and it is robust to possible future changes (& → | so the constraints have the same meaning everywhere...)

"async" {< "111.25.00"}
"async" {>= "113.00.00"}